packages/mobile/src/components/core/KeyboardAvoidingView.tsx

```diff
@@ -20,6 +20,8 @@ type KeyboardAvoidingViewProps = {
   keyboardHidingDuration?: number
   // Offset is subtracted from the desired height when the keyboard is showing.
   keyboardShowingOffset?: number
+  onKeyboardShowCallback?: () => void
```

Comment:
nit: 'Callback' is redundant? I feel like the normal pattern is `onEvent` as a prop and `handleEvent` as the handler